src/towncrier/_project.py

```diff
@@ -38,47 +38,45 @@ def _get_package(package_dir: str, package: str) -> ModuleType:
 
 
 def get_version(package_dir: str, package: str) -> str:
-    module = _get_package(package_dir, package)
+    version: Any
 
+    module = _get_package(package_dir, package)
     version = getattr(module, "__version__", None)
-
     if not version:
-        raise Exception("No __version__, I don't know how else to look")
+        try:
```

Comment:
it looks like this is not only about removing incremental, but also about using `importlib_metadata` to get the version of  the target project

Reply:
redundant as linked PR has been merged. Updated and integrated latest trunk in  #627